### PR TITLE
Interchain Security roadmap

### PR DESCRIPTION
**Pull request by mpoke**

### CODE REVIEW

File: docs/ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ 
+## Q2 - 2022
+
+- Finalize core ICS features that enable a provider chain to provide security to multiple consumer chains.
+  - https://github.com/cosmos/interchain-security/issues/35
+  - https://github.com/cosmos/interchain-security/issues/27
+  - https://github.com/cosmos/interchain-security/issues/39
+- Create a minimum viable consumer chain (MVCC) that enables testing core ICS features.
+  - https://github.com/cosmos/interchain-security/issues/139
+- Create in-memory test drive based on the ibc-go testing framework.
```
Comment (jtremback) [inline docs/ROADMAP.md:15]:
```suggestion
- Create in-memory test driver based on the ibc-go testing framework.
```

File: docs/ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- Create a CosmWasm-enabled consumer chain.
+  - https://github.com/cosmos/interchain-security/issues/143
+- Enable validators to set different consensus keys for different consumer chains.
+  - https://github.com/cosmos/interchain-security/issues/26
+- Enable the chains to be easily restarted in case they halt.
+  - https://github.com/cosmos/interchain-security/issues/158
+  - https://github.com/cosmos/interchain-security/issues/156
+  - https://github.com/cosmos/interchain-security/issues/125
+  - https://github.com/cosmos/interchain-security/issues/121
+- Fix remaining issues
```
Comment (jtremback) [inline docs/ROADMAP.md:36]: I would move anything relating to ICS on the Hub up to the top of the priority list

Comment (jtremback) [reply docs/ROADMAP.md:36]: Not that this list is necessarily in order of priority, but move it up still

File: docs/ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  - https://github.com/cosmos/interchain-security/issues
+- Test edge cases regarding channel initialization, consumer chain removal, and clients expiring.
```
Comment (jtremback) [inline docs/ROADMAP.md:38]: Again, move up ICS on the provider stuff

File: docs/ROADMAP.md (hunk 2 above)
Comment (jtremback) [reply docs/ROADMAP.md:36]: It looks like everything past this point is ICS on the provider, so I would basically just flip the two halves of the list

**Review by sainoe — APPROVED** [review]
Good Work @mpoke !
